### PR TITLE
New custom display font. Overhauled formatting, CSS, and colors of ECAM pages

### PR DESCRIPTION
**Pull request by wpine215**
## Summary of Changes

- Imported new custom font for all displays, originally made by JayZee and modified by me
- Cleaned up existing CSS code for ECAM pages and created shared CSS file to define common attributes and colors

## Screenshots (if necessary)

Before:
95009703-1b1c5c00-05f2-11eb-83fe-462430b3c007

After:
FlightSimulator_gaYBnmT29w

## References

#reference-docs on Discord as well as reference photos on the Google drive.

## Additional context

The oversized compass legends and cramped altimeter text have been fixed in the JS, but aren't visible in the screenshot because the hot-reload did not pick up on the JS modifications.

Discord username (if different from GitHub): Iceman#5540

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build.py will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, slick on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by wpine215** [comment]
Fixed inconsistencies in PFD/ND/ECAM. Added support for MCDU and ISIS.

Updated screenshots:
FlightSimulator_wdut6boVfr

FlightSimulator_AUXKuDuAeQ

FlightSimulator_hLmI02tLZm

**Comment by tareksabet** [comment]
I don't know if you intentionally did that or not but the squares in the INIT A page shouldn't be red it should be in orange

**Comment by tyler58546** [comment]
The MCDU should have 2 font sizes, but everything is the same size in your screenshot.

**Comment by wpine215** [comment]
> The MCDU should have 2 font sizes, but everything is the same size in your screenshot.

I'll have to tweak the smaller font class size then, thanks for pointing it out.

> I don't know if you intentionally did that or not but the squares in the INIT A page shouldn't be red it should be in orange

I didn't touch the actual color classes (e.g. red, yellow, etc) in the MCDU CSS, apart from fine-tuning the individual colors. I'll have to go through the MCDU code and fix the color assignments in a separate PR.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css [hunk 1]
```diff
@@ -102,22 +105,22 @@ a320-neo-cdu-main-display::after {
...
-  color: #00ff00; }
+  color: var(--displayGreen); }
 
 .red {
-  color: #ff9a00; }
+  color: var(--displayRed); }
 
 .magenta {
-  color: #f800cc; }
+  color: var(--displayMagenta); }
```
Comment (tyler58546) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css:123]: I don't think these should have been changed, we already fixed these colors in #888.

Comment (wpine215) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU.css]: Reverted them to the original colors, put the color definitions in the new common CSS file.

**Comment by wpine215** [comment]
New preview of MCDU with original (unchanged) colors, and font size fix:

FlightSimulator_R9xJYZMk3u

**Comment by tyler58546** [comment]
Looks much better, but the small font should spaced such that it has the same width as the large font (so a message that is 5 characters long should be same width in both sizes)

**Comment by wpine215** [comment]
> 
> 
> Looks much better, but the small font should spaced such that it has the same width as the large font (so a message that is 5 characters long should be same width in both sizes)

Unfortunately the issue I'm running into on the T/O PERF page is that the center small text will overlap with the right-hand text, I'll need to figure out a way to fix the alignment in a future PR, and then increase the font spacing then as well.

**Pull request merged**